jsonschema-core.xml

```diff
@@ -2430,7 +2430,7 @@
                         </t>
                         <t>
                             The annotation result of this keyword is the set of instance
-                            property names matched by this keyword.
+                            property names matched by at least one key in this keyword.
```

Comment:
Do we say "key" elsewhere? I think we generally say "property", as the thingy underneath a JSON-object.
Also "in" -> "under" as above.

```suggestion
                            property names matched by at least one property under this keyword.
```